src/usecases/verifyTrustAdminToken.test.js

```diff
@@ -65,4 +67,55 @@ describe("verifyTrustAdminToken", () => {
       Location: "/trust-admin/login",
     });
   });
+
+  it("updates the cookie if the token is regenerated", () => {
+    const regenRes = {
+      writeHead: jest.fn(() => regenRes),
```

Comment:
I don't quite understand this. We create a response `regenRes` with a method `writeHead` with a stubbed response to return itself? 🤔

Reply:
It's to support the following chain at the end of the usecase:
```
 res.writeHead(302, { Location: "/admin/login" }).end();
```
We call writeHead on the res object and then call end() on the res returned by writeHead. So to simulate that the writeHead mock needs to return the res.

Reply:
Ah I see that now, thanks!